examples/pymc3_howto/profiling.ipynb

```diff
@@ -31,16 +31,6 @@
     "print(f\"Running on PyMC3 v{pm.__version__}\")"
```

Comment:
<p>Should be <code>theano,xarray</code>.  And you need to rerun the notebook for the changes to be reflected here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>